src/components/dashboard-widgets/dashboard-actions-widget/dashboard-actions-widget.component.html

```diff
@@ -0,0 +1,28 @@
+<ux-predefined-widget class="widget-element"
+                      [id]="id"
+                      [colSpan]="colSpan"
+                      [rowSpan]="rowSpan"
+                      [name]="name"
+                      [heading]="heading"
+                      [fixedMode]="fixedMode">
+    <div class="widget-actions-wrapper">
+        <div class="widget-status">
+            <span class="widget-status-icon" *ngIf="status.icon || status.iconTemplate">
+                <ux-icon *ngIf="status.icon" name="{{ status.icon }}"></ux-icon>
```

Comment:
Bindings should be used rather than interpolation within attributes:

```suggestion
                <ux-icon *ngIf="status.icon" [name]="status.icon"></ux-icon>
```

There are a few other occurrences of this